### PR TITLE
LRQA-67916 - Create test stubs for All Items (List View) tests

### PR DESCRIPTION
**Pull request by leticiachiu**

### CODE REVIEW

**Comment by leticiachiu** [comment]
cc @anabuchmann

**Comment by rodrigocunhaa** [comment]
ci:test:sf

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase [hunk 1]
```diff
@@ -0,0 +1,455 @@
...
+
+	@description = "LPS-92053 - Verify that, in the item detail modal, Open when an SLA is paused, the SLA status will be related to the moment when it triggered the pause timer"
+	@ignore = "Test Stub"
+	@priority = "4"
+	test WhenSlaIsPausedSlaStatusIsRelatedToThePauseTimer {
+
+		// TODO LRQA-67767 WhenSlaIsPausedSlaStatusIsRelatedToThePauseTimer pending implementation
+
+	}
+
```
Comment (anabuchmann) [inline modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:454]: Hi @leticiachiu !
There is one stub missing. Could you, please, add it?
```
@description = "LPS-87571 - Verify that the filter query only be applied after user moves the cursor out of the dropdown cardr"	
@ignore = "Test Stub"	
@priority = "3"	test FilterIsAppliedAfterCursorMovesOut {		
// TODO LRQA-67773 FilterIsAppliedAfterCursorMovesOut pending implementation	}
```
You can find it in line 603 of the Test Map.

**Review by anabuchmann — CHANGES_REQUESTED** [review]
Hi @leticiachiu !
Could you, please, see the comments and make the chances I suggested?
Thank you!!

File: modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase (hunk 1 above)
Comment (leticiachiu) [reply modules/apps/portal-workflow/portal-workflow-test/src/testFunctional/tests/WorkflowMetricsAllItemsList.testcase:454]: hello @anabuchmann! you can find this stub on line 158 :)

**Comment by leticiachiu** [comment]
fixed @anabuchmann

**Comment by rodrigocunhaa** [comment]
Just started reviewing :)

**Comment by rodrigocunhaa** [comment]
ci:test:sf

**Comment by rodrigocunhaa** [comment]
ci:forward

**Pull request closed without merging**